pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[tool.poetry]
```

Comment:
drop?

Reply:
Should be kept since this is a poetry config file for people using the Poetry file manager